lib/src/chart/base/base_chart/render_base_chart.dart

```diff
@@ -50,22 +52,20 @@ abstract class RenderBaseChart<R extends BaseTouchResponse> extends RenderBox
 
   /// Initializes our recognizers and implement their callbacks.
   void initGestureRecognizers() {
-    _panGestureRecognizer = PanGestureRecognizer();
-    _panGestureRecognizer
-      ..onDown = (dragDownDetails) {
-        _notifyTouchEvent(FlPanDownEvent(dragDownDetails));
+    _scaleGestureRecognizer = ScaleGestureRecognizer();
+    _scaleGestureRecognizer
+      ..onStart = (details) {
+        _notifyScaleEvent(FlScaleStartEvent(details));
```

Comment:
Does it get called for the second finger (when you already have your first finger, and try to add the second finger to do the scale gesture)?

Reply:
In this case, I think we should have something like fingerIndex or something like that